x/gobenchdata-web/main.go

```diff
@@ -18,7 +18,8 @@ var (
 	source    = pflag.StringP("source", "s", "", "source repository for package, eg 'github.com/bobheadxi/gobenchdata'")
 	canonical = pflag.StringP("canonical-import", "i", "", "canonical import path for package, eg 'go.bobheadxi.dev/gobenchdata' (used to generate links to source code)")
 
-	chartsTypes = pflag.StringP("charts-types", "c", "bytes/op,allocs/op", "additional chart types to generate (comma-separated, in addition to 'ns/op')")
+	chartsTypes  = pflag.StringP("charts-types", "c", "bytes/op,allocs/op", "additional chart types to generate (comma-separated, in addition to 'ns/op')")
+	perBenchmark = pflag.BoolP("per-benchmark", "p", false, "open to create a graph per benchmark. Default is false which will group benchmarks by package")
```

Comment:
This is a pretty granular option, so might be good to just enforce the long-hand version of the flag (`-p` also has other connotations usually, like port):

```suggestion
	perBenchmark = pflag.Bool("per-benchmark", false, "toggle per-benchmark graphs (grouped by package by default)")
```

^ also included a slight rewording to the docstring for consistency with the others